setupPlugins.js

```diff
@@ -0,0 +1,3 @@
+// This enables ESLint to use dependencies of this config
+// (see https://github.com/eslint/eslint/issues/3458)
+require('@rushstack/eslint-patch/modern-module-resolution');
```

Comment:
ESLint wants to handle this itself one day, but in the meantime there are some good guys from Microsoft providing a patch.